client/python/examples/README.md

```diff
@@ -0,0 +1,246 @@
+# Python Client Examples
+
+Currently there are five example files:
+
+- [simple.py](#simple.py)
+- [general.py](#general.py)
+- [queues.py](#queuespy)
+- [cancelling.py](#cancellingpy)
+- [monitor.py](#monitor.py)
+
+There is also a section on [Using Basic Auth](#using-basic-auth)
+
+## Setup
+
+Please see the [python client docs](https://github.com/G-Research/armada/blob/master/client/python/README.md) for getting the client setup.
+
+### Running the examples
+
+> Each example has three enviromental variables for setup
+> ```bash
+> export ARMADA_SERVER=localhost
+> export ARMADA_PORT=443
+> export DISABLE_SSL=true
+>```
+
+> You can then simply run one of the examples
+> ```bash
+> python3 simple.py
+> ```
+
+## Example Files
+
+### simple.py
+
+[Link to File](https://github.com/G-Research/armada/tree/master/client/python/examples/simple.py)
+
+Example of using the Armada client to create a queue, jobset and job,
+then watch for the job to succeed or fail.
+
+#### Walkthrough
+
+> Armada uses Kubernetes to control jobs, so we need to define a podspec that our jobs will use.
+> ```py
+> def create_dummy_job(client: ArmadaClient):
+> ```
+
+> If you are using a remote server, secure_channel should be used to ensure that all infomation is encrypted.
+> ```py
+> if DISABLE_SSL:
+>     channel = grpc.insecure_channel(f"{HOST}:{PORT}")
+> else:
+>     channel_credentials = grpc.ssl_channel_credentials()
+>     channel = grpc.secure_channel(
+>         f"{HOST}:{PORT}",
+>         channel_credentials,
+>     )
+> ```
+
+
+> All queues need a priority factor to be created, which is used to determine the order in which jobs are run.
+> ```py
+> client.create_queue(name=queue, priority_factor=1)
+> ```
+
+
+### general.py
+
+[Link to File](https://github.com/G-Research/armada/tree/master/client/python/examples/general.py)
```

Comment:
These links don't work if you try to view the markdown.  Maybe you should just use a relative link and not the branch name?

Reply:
They won't work until the code is merged, but I can use relative links now